tests/test_metrics.py

```diff
@@ -22,30 +22,30 @@ def setUp(self):
             with open("metric", "w+") as fd:
                 fd.write(branch)
 
-            with open("metric_json", "w+") as fd:
+            with open("metric.json", "w+") as fd:
```

Comment:
Let's not touch all of these tests, as they mostly test explicit `-t` notation. Let's instead add a new test that would test that type gets properly detected from the extension.